lib/roadie/inliner.rb

```diff
@@ -96,7 +100,12 @@ def elements_matching_selector(stylesheet, selector)
       nil
     end
 
-    def add_uninlinable_styles(parent, blocks)
+    # Adds unlineable styles in the specified part of the document
+    # either the head or in the document
+    # @param [Symbol] parent  Where to put the styles
+    # @param [Array<StyleBlock>] blocks  Non-inlineable style blocks
+    # @param [Boolean]  use_shared_media_queries  Whether to group media queries
```

Comment:
Nice touch. This is a private method so documentation is not needed, but it doesn't hurt either. :)